IanGRap/Squish
# Reduce Death Particles

Contributors: Austin

Goal: Tame performance issues by reducing the number of particles spawned on player death

How to Test:
- Load up the game and make sure particles still look good
- If there is a performance boost, this is good :)
